ghost/core/core/server/services/email-service/EmailService.js

```diff
@@ -121,6 +126,10 @@ class EmailService {
         const emailCount = await this.#emailSegmenter.getMembersCount(newsletter, emailRecipientFilter);
         await this.checkLimits(emailCount);
 
+        const csdEmailCount = this.#domainWarmingService.isEnabled()
+            ? await this.#domainWarmingService.getWarmupLimit(emailCount)
+            : undefined;
```

Comment:
Do I remember correctly that we use `undefined` here to not intentionally set `0` in order to distinguish between already used (even with 0 sends) and not used at all? Maybe worth a comment here, to remember this context.